blog/content/post/2019-06-03-Hummingboard-Pulse-first-impression.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Hummingboard Pulse - first impression.
+abstract: In this post, we will take a look at one of the SolidRun product -
+          the HummingBoard Pulse. After power up the board we will try to boot
+          operating system on it.
+cover: /covers/hummboard.jpg
+author: tomasz.zyjewski
+layout: post
+published: false
+date: 2019-06-10
+archives: "2019"
+
+tags:
+  - HummingboardPulse
+  - SOM
+  - i.MX8
+  - u-boot
+  - linux
+categories:
+  - Firmware
+
+---
+
+### Introduction
+
+SolidRun is a global leading developer of embedded systems and he provides
+a lot of powerful products. One of them I want to describe in this post and
+it is the HummingBoard Pulse. You can see it in the picture below.
+
+![HummingBoard picture](/img/hummboard.jpg)
+
+Hummingboard Pulse house the powerful i.MX8M SOM with a quad or dual-core ARM
+Cortex A53 processor (up to 1.5 GHz) with ARM M4. The i.MX8M family of processors
+provides industry-leading audio, voice and video processing for applications that
+scale from consumer home audio to industrial building automation and mobile computers.
+More information can be found at the official website of the producer of this
+processors [there](https://www.nxp.com/products/processors-and-microcontrollers/arm-based-processors-and-mcus/i.mx-applications-processors/i.mx-8-processors/i.mx-8m-family-armcortex-a53-cortex-m4-audio-voice-video:i.MX8M)
+
+The HummingBoard Pulse provides a lot of hardware interfaces from which I can
+mention:
+* USB type C
+* Micro USB
+* RJ45 Ethernet
+* 2x USB 3.0
+* HDMI 2.0
+* Audio Headset
+* MicroSD
+
+All of them are nicely shown on the pictures below.
+
+This is the front of the board.
+
+![HummingboardPulse front layout](/img/HummingboardBoard-Pulse-front-layout.png)
+
+And there we have interfaces located on the back of the board.
+
+![HummingboardPulse back layout](/img/HummingboardBoard-Pulse-back-layout.png)
+
+### Getting started with the board.
+
+To start using the HummingBoard Pulse you will need a couple of things:
+* Linux or Windows PC (it will be easier if you'll have Linux PC),
+* 16GB Micro SD card,
+* 12V Power adapter (the board has wide range input of 7V-36V but 12V is recommended),
+* MicroUSB to USB for the console because the HummingBoard Pulse has an onboard FTDI chip,
```

Comment:
you can add that there is no need to use an external UART/USB converter because of this.